mondey_backend/src/mondey_backend/import_data/import_children_with_assigned_milestone_data.py

```diff
@@ -0,0 +1,214 @@
+            ).scalar_one_or_none()
+
+            print(row["FK01"])
+            print("And month:")
+            print(row["FK02"])
+
+            # Hardcoded
+            birth_year_mapping = {
+                9: 2025,
+                1: 2024,
+                2: 2023,
+                3: 2022,
+                4: 2021,
+                5: 2020,
+                6: 2019,
+                7: 2018,
+                8: 2017,  # Converting "Vor 2018" to 2017
+            }
+
+            birth_month_mapping = {
+                1: 1,  # Januar
+                2: 2,  # Februar
+                3: 3,  # März
+                4: 4,  # April
+                5: 5,  # Mai
+                6: 6,  # Juni
+                7: 7,  # Juli
+                8: 8,  # August
+                9: 9,  # September
+                10: 10,  # Oktober
+                11: 11,  # November
+                12: 12,  # Dezember
+            }
+
+            if not existing_child:
+                # Make the parent first, for parent questions and milestone answering sessions.
+                parents_id = parent_id_map[
+                    str(child_id)
+                ]  # still works: get_childs_parent_id(child_id)
+
+                # Create a new child
+                child = Child(
+                    id=child_id,
+                    name=f"Imported Child {child_id}",
+                    birth_year=birth_year_mapping[int(row["FK01"])],  # FK01, mapped...
+                    birth_month=birth_month_mapping[int(row["FK02"])],
+                    user_id=parents_id,
+                    has_image=False,
+                )
+                session.add(child)
+                session.commit()
+                print(f"Created child with ID: {child_id}")
+            else:
+                # @liam this supports updating with future milestone answer sessions for the same children, but,
+                # we probably want to know from the researchers whether to overwrite or add new milestones, especially
+                # what to do when the milestone level is the same as existing data (i.e. probably duplicate mistake)
+                # So for now it will raise a ValueError to prevent messing up the data on import, to be safe.
+                child = existing_child
+                print(f"Using existing child with ID: {child_id}")
+                raise ValueError(
+                    "There appears to be duplicate data for a child - they already have data. "
+                    "Are you sure you want to add further milestone data, possibly duplicating answers?"
+                )
+
+            # Create a milestone answer session for this child
+            answer_session = MilestoneAnswerSession(
+                child_id=child_id,
+                user_id=parents_id,
+                expired=True,
+                included_in_statistics=False,
+                created_at=datetime(2025, 1, 1, 1, 0, 1),
```

Comment:
I suggested making this date 2025/01/01 but I just realised this is a very bad idea! When calculating the stats we determine how old the child was when they answered the questions for an answersession based on their date of birth and when the answer session was created. So this date really needs to be the (approx) date when the questionnaire data was collected - is that stored somewhere in the data? (by the way this would also explain why I am seeing negative child ages in the research page data)

Reply:
@jmsssc looks like there's a `STARTED` column with the date we need for this - otherwise this data all looks great from playing around with it locally!

Reply:
I updated the mondey.db you sent me using these STARTED values with this notebook: 5c2feb6